diesel/src/query_builder/debug.rs

```diff
@@ -0,0 +1,35 @@
+use super::{QueryBuilder, BuildQueryResult};
+use types::NativeSqlType;
+
+#[doc(hidden)]
+pub struct DebugQueryBuilder {
```

Comment:
@sgrif How do you want to handle docs for internals like this.  Is doc #[doc(hidden)] fine or should there hidden docs for contributors only?